solutions/standard/variables.tf

```diff
@@ -354,3 +354,9 @@ variable "elasticsearch_full_version" {
   type        = string
   default     = null
 }
+
+variable "kibana_image_reference" {
+  description = "The docker image reference for Kibana.Please give your image input or leave blank for default."
```

Comment:
The docker image reference to use for Kibana if `enable_kibana_dashboard` is set to true. If no value is set, it will pull the image from the official Elastic registry (https://www.docker.elastic.co). Ensure to use a version that is compatible with the Elasticsearch version being used.

Reply:
Changed the description.